invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/theme.js

```diff
@@ -134,13 +134,15 @@ const handleAuthButtonClick = () => {
 
 $authButton.on({ click: handleAuthButtonClick });
 
-const invenioConfig = JSON.parse(document.body.dataset.invenioConfig);
-const isMathJaxEnabled = invenioConfig?.isMathJaxEnabled;
-if (window.invenio) {
-  window.invenio.onSearchResultsRendered = () => {
-    if (isMathJaxEnabled) {
-      // Re-render mathematical content on the page using MathJax.
-      return window.MathJax?.typeset();
-    }
-  };
+if (document.body.dataset.invenioConfig) {
```

Comment:
This should always be there. @anikachurilova worked on this.
What was the issue?

Reply:
it is not there in the frontpage

Reply:
Then it should be fixed. @anikachurilova can you please check?